optima/results.py

```diff
@@ -100,6 +101,46 @@ def __repr__(self):
         output += '============================================================\n'
         output += objrepr(self)
         return output
+        
+    
+    def __add__(self, R2):
+        ''' Define how to add two Resultsets '''
+        if type(R2)!=Resultset: raise OptimaException('Can only add two results sets')
```

Comment:
i think you can actually add arbitrarily many resultsets, can't you? but you can only add a resultset to a resultset